File: Aero Linea/Aerolinea.API/Aerolinea.API/Models/Loginresponsedto.cs
namespace Aerolinea.API.DTOs
{
    /// <summary>
    /// DTO de respuesta devuelto al cliente despues de un inicio de sesion exitoso.
    /// Contiene identificacion del usuario, nombre, correo y datos del rol asignado.
    /// </summary>
    public class LoginResponseDto
    {
        public int UsuarioId { get; set; }
        public string Nombre { get; set; }
        public string Correo { get; set; }
        public int RolId { get; set; }
        public string RolNombre { get; set; }
    }
}

File: Cadena de Hoteles/Hotel/src/main/java/org/example/dtos/LoginResponseDTO.java
package org.example.dtos;

/**
 * DTO con los datos retornados al cliente tras una autenticacion exitosa.
 * Incluye el mensaje de confirmacion, el username y el rol del usuario autenticado.
 */
public class LoginResponseDTO {

    private String mensaje;
    private String username;
    private int    rolId;

    /**
     * Constructor que inicializa la respuesta de login con todos sus datos.
     * @param mensaje  mensaje de confirmacion de la autenticacion.
     * @param username nombre de usuario autenticado.
     * @param rolId    ID del rol asignado al usuario.
     */
    public LoginResponseDTO(String mensaje, String username, int rolId) {
        this.mensaje   = mensaje;
        this.username  = username;
        this.rolId     = rolId;
    }

    /**
     * Retorna el mensaje de confirmacion de la autenticacion.
     * @return mensaje de confirmacion.
     */
    public String getMensaje() { return mensaje; }

    /**
     * Retorna el nombre de usuario autenticado.
     * @return username del usuario.
     */
    public String getUsername() { return username; }

    /**
     * Retorna el ID del rol asignado al usuario autenticado.
     * @return ID del rol.
     */
    public int getRolId() { return rolId; }
}